pkg/deployments/tasks/20210727-meta-stable-pool/output/kovan.json

```diff
@@ -0,0 +1,5 @@
+{
+  "query": "0x88919A6C36359d449322Cd40451749553C6E33A2",
```

Comment:
Seems these are not verified?

Reply:
Yup we need to tweak the verification process to support libraries 😬